src/jgikbase/idmapping/core/HashedToken.py

```diff
@@ -0,0 +1,17 @@
+'''
```

Comment:
Style nitpicks:
- You are using more java-styled directory structure. In python, a directory is typically a package containing modules. So it doesn't make sense to have packages named `src`, `jgikbase`, or `core`. Typically there is just one directory called `idmapping` (name of your actual package). See here: http://docs.python-guide.org/en/latest/writing/structure/
- You might consider naming your modules lowercase with underscores, as per pep: https://www.python.org/dev/peps/pep-0008/#package-and-module-names
- Pep also calls for double-quote docstrings: https://www.python.org/dev/peps/pep-0257/
- In python 3 code, I always see classes as `class MyClass:` without the extra `(object)` which isn't necessary